pennylane/transforms/__init__.py

```diff
@@ -167,10 +137,6 @@
 .. autosummary::
     :toctree: api
 
-    ~single_tape_transform
-    ~batch_transform
-    ~qfunc_transform
-    ~op_transform
```

Comment:
Please could we move this to a section at the bottom of everything:

```
Old transforms framework
------------------------

These utility functions were previously used to create transforms in PennyLane and will be
deprecated soon. It is now recommended to use :class:`qml.transform <pennylane.transform>`
for creation of custom transforms.

.. autosummary::
    :toctree: api

    ~single_tape_transform
    ~batch_transform
    ~qfunc_transform
    ~op_transform

Reply:
Done!